airflow/config_templates/default_airflow.cfg

```diff
@@ -133,6 +133,11 @@ load_default_connections = True
 # Where your Airflow plugins are stored
 plugins_folder = {AIRFLOW_HOME}/plugins
 
+# Should tasks be executed via forking of the parent process ("False",
+# the speedier option) or by spawning a new python process ("True" slow,
+# but means plugin changes picked up by tasks straight away)
+# Default: execute_tasks_new_python_interpreter = False
+
```

Comment:
This looks a bit odd, why not just have it as:

```ini
execute_tasks_new_python_interpreter = False
```

?

Reply:
Or atleast:

```
# execute_tasks_new_python_interpreter = False
```

The `Default:` in the front of `key = val` might make users search for the config below the comments, at least a the first glance.

Reply:
I guess I shouldn't start a new convention here, but this was driven by the sort of change in https://github.com/apache/airflow/pull/11322. I'll pull this out.